apps/api/src/server/migration/migrators/s51-advice-migrator.js

```diff
@@ -35,10 +37,20 @@ export const migrateS51Advice = async (s51AdviceList) => {
 			databaseConnector.$executeRawUnsafe(s51AdviceStatement, ...s51AdviceParameters)
 		]);
 
-		// TODO attachments
-		// for (const attachmentId in s51AdviceEntity.attachmentIds) {
-		// 	// create attachment/document
-		// }
+		await Promise.all(
+			s51Advice.attachmentIds.map(async (attachmentId) => {
+				return documentRepository.getById(attachmentId).then((documentRow) => {
+					if (!documentRow || !documentRow.guid) {
+						return Promise.reject(new Error(`Failed to get document with ID ${attachmentId}`));
```

Comment:
Do we need to explicitly reject when we are already throwing error

Reply:
good spot. Fixed